packages/collector/test/tracing/cloud/aws/lambda/app.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2021
+ */
+
+'use strict';
+
+require('../../../../../src')();
+
+const AWS = require('aws-sdk');
+const express = require('express');
+const logPrefix = `AWS Lambda (${process.pid}):\t`;
+AWS.config.update({ region: 'us-east-2' });
+const lambda = new AWS.Lambda();
+const functionName = process.env.AWS_LAMBDA_FUNCTION_NAME || 'team-nodejs-invoke-function';
+const log = require('@instana/core/test/test_util/log').getLogger(logPrefix);
+
+const operations = {
+  invoke: {
+    FunctionName: functionName,
+    // "Event"|"RequestResponse"|"DryRun"
+    InvocationType: 'RequestResponse',
+    Payload: '{"ok": true}'
+  },
+  invokeAsync: {
+    FunctionName: functionName,
+    InvokeArgs: '{"ok": true}'
+  }
+};
+
+/**
+ * @param {'invoke' | 'invokeAsync'} op
+ * @param {(err: *, data: *) => void} cb
+ * @param {boolean} withError
+ * @param {boolean} ctx
+ */
+function execOperation(op, cb, withError = false, ctx = false) {
+  /** @type {import('aws-sdk').Lambda.Types.InvocationRequest | import('aws-sdk').Lambda.Types.InvokeAsyncRequest} */
+
+  const options = { ...operations[op] };
+  if (withError) {
+    options.InvalidParameter = 1;
+  }
+
+  if (ctx && op === 'invoke') {
+    options.ClientContext = 'eyJzb21lX2NvbnRlbnQiOiAxfQ==';
```

Comment:
Just for my understanding: What is encoded in this client context in the test application? Ah, nevermind, it is just a test value "{"some_content": 1}". Maybe you could spell that out in clear text and then encode it to base64 on the fly, so it becomes obvious? (This relates to another comment around client context further down.)